teacher.rb

```diff
@@ -0,0 +1,14 @@
+require_relative('./person')
+
+class Teacher < Person
+  attr_accessor :specialization
+
+  def initialize(name, age, parent_permission, specialization)
```

Comment:
- [ ] Great job implementing the `Teacher` class 👍 however according to the requirements `name` and `parent_permission` should have default values of `"Unknown"` and `true`.

Reply:
Thank you Mikethreels

Reply:
Fixed, Thanks